### PR TITLE
Fixing ngram sh

### PR DESCRIPTION
**Pull request by look-luc**
When I tried to run the sbatch run_clearlab2.sh experiments/ngram.cfg -o language_code=..., in the folder of the bash, it will not properly run so I just added a line to go up a directory to the run.py before running it

### CODE REVIEW

File: scripts/run_clearlab2.sh [hunk 1]
```diff
@@ -2,14 +2,17 @@
...
-#SBATCH --mem=4000m
+#SBATCH --mem=32G
 #SBATCH --time=1:00:00
 #SBATCH --output=logs/%j.log
 #SBATCH --job-name=specdec
 #SBATCH --partition=blanca-clearlab2
 #SBATCH --account=blanca-clearlab2
 #SBATCH --qos=blanca-clearlab2
 
+export HF_HOME="/projects/lude4390/.cache/huggingface"
```
Comment (michaelpginn) [inline scripts/run_clearlab2.sh]: Instead of using your username, can you use `$USER` so it work for anyone?

File: scripts/run_clearlab2.sh [hunk 2]
```diff
@@ -2,14 +2,17 @@
 #SBATCH --gres=gpu:1
 #SBATCH --ntasks=1
 #SBATCH --cpus-per-task=2
-#SBATCH --mem=4000m
+#SBATCH --mem=32G
```
Comment (michaelpginn) [inline scripts/run_clearlab2.sh:5]: Are you seeing OOMs?

File: scripts/run_clearlab2.sh [hunk 3]
```diff
@@ -25,4 +28,6 @@ if torch.cuda.is_available():
     print("GPU 0:", torch.cuda.get_device_name(0))
 PY
 
+cd ..
```
Comment (michaelpginn) [inline scripts/run_clearlab2.sh:31]: This assumes you run from the `scripts/` directory? To be consistent, does it work to just submit the bash script from the main directory:

```bash
sbatch scripts/run_clearlab2.sh ...
```

Not a huge issue, but my personal preference is to avoid moving around in batch scripts if possible.

**Review by michaelpginn — CHANGES_REQUESTED** [review]

File: scripts/run_clearlab2.sh (hunk 2 above)
Comment (look-luc) [reply scripts/run_clearlab2.sh:5]: I was yes

File: scripts/run_clearlab2.sh (hunk 3 above)
Comment (look-luc) [reply scripts/run_clearlab2.sh:31]: I tried doing this and (at least for me) it does not like it so that's why I put it

File: scripts/run_clearlab2.sh (hunk 2 above)
Comment (michaelpginn) [reply scripts/run_clearlab2.sh:5]: Was it CUDA OOM? I'd be surprised if we're running out of CPU RAM, since the large stuff (logits during spec dec) is all on GPU.

If it was a CPU OOM, do you have a rough idea of how much memory you need? 32 seems a bit high and you might have other people complain if it blocks their jobs.

Comment (look-luc) [reply scripts/run_clearlab2.sh:5]: I don't remember exactly what it was but it could well be that it could only need 8000M.  Maybe just to be safe for the data that have a fair number of entries, there should be some grace so that we won't run to this OOM down the line.

**Review by michaelpginn — APPROVED** [review]
Need to fix merge conflicts, then merge
